src/cleaner/__init__.py

```diff
@@ -55,6 +55,20 @@ def load_json_file(self, file_path: str) -> dict:
             logging.error(f"Error loading file at {file_path}: {e}")
             return {}
 
+    def remove_judicial_officer(self, data):
+        # Check if data is a dictionary
+        if isinstance(data, dict):
+            # Remove 'judicial officer' if it exists in this dictionary
+            if "judicial officer" in data:
+                del data["judicial officer"]
```

Comment:
🔧 we should add a doc comment for this method.  I can't recall why we need to delete the JO information.